### ISSUE TITLE
Starting OTD with Windows Ink mode crashes the program on startup

### ISSUE THREAD

**Issue by PuzThePuzzle** [X9VoiD/VoiDPlugins#29]
Im trying to install it once again, but it doesnt want to boot up if windows ink (absolute mode) crashes the entire program

**Comment by PuzThePuzzle** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/75788999/154814243-85950469-b2b1-43e7-9b18-e3a50197a37b.png)
![image](https://user-images.githubusercontent.com/75788999/154814246-b4de32dc-dc2b-4065-a219-25d418d19d79.png)

**Comment by DaniJVaz** [comment]
Same problem here: running OTD with a Huion H1161 in Windows Ink Absolute Mode immediately crashes the daemon and quits. Running the daemon on its own reports the following error:

```
Unhandled exception. System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at VoiDPlugins.Library.VoiD.GlobalStore`1.GetOrInitialize[U](TabletReference tabletReference, Func`1 initialValue) in /home/runner/work/VoiDPlugins/VoiDPlugins/src/VoiDPlugins.Library/VoiD/GlobalStore.cs:line 21
   at VoiDPlugins.OutputMode.WinInkBasePointer..ctor(String name, TabletReference tabletReference) in /home/runner/work/VoiDPlugins/VoiDPlugins/src/OutputMode/WindowsInk/Pointer/WinInkBasePointer.cs:line 21
   at VoiDPlugins.OutputMode.WinInkAbsolutePointer..ctor(TabletReference tabletReference, IVirtualScreen screen) in /home/runner/work/VoiDPlugins/VoiDPlugins/src/OutputMode/WindowsInk/Pointer/WinInkAbsolutePointer.cs:line 12
   at VoiDPlugins.OutputMode.WinInkAbsoluteMode.Initialize() in /home/runner/work/VoiDPlugins/VoiDPlugins/src/OutputMode/WindowsInk/Output/WinInkAbsoluteMode.cs:line 25
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at OpenTabletDriver.Desktop.Reflection.PluginSettingStore.TriggerEventMethods(Object obj, TabletReference tabletReference)
   at OpenTabletDriver.Desktop.Reflection.PluginSettingStore.Construct[T](TabletReference tabletReference, Boolean trigger)
   at OpenTabletDriver.Daemon.DriverDaemon.SetSettings(Settings settings)
   at OpenTabletDriver.Daemon.DriverDaemon.LoadUserSettings()
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_1(Object state)
   at System.Threading.QueueUserWorkItemCallbackDefaultContext.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()
   at System.Threading.Thread.StartCallback()
```

**Comment by X9VoiD** [comment]
Fixed by 10b4b008232f1bfea3711473f882864c13ede151
